src/utils.ts

```diff
@@ -1,5 +1,47 @@
 import React from 'react';
 
+const HTML_ENTITIES: Record<string, string> = {
+  '&amp;': '&',
+  '&lt;': '<',
+  '&gt;': '>',
+  '&quot;': '"',
+  '&#39;': '\'',
+  '&apos;': '\'',
+  '&nbsp;': ' ',
+  '&copy;': '©',
+  '&reg;': '®',
+  '&deg;': '°',
+  '&plusmn;': '±',
+  '&para;': '¶',
+  '&middot;': '·',
+  '&ndash;': '\u2013',
+  '&mdash;': '\u2014',
+  '&lsquo;': '\u2018',
+  '&rsquo;': '\u2019',
+  '&sbquo;': '\u201A',
+  '&ldquo;': '\u201C',
+  '&rdquo;': '\u201D',
+  '&bdquo;': '\u201E',
+  '&hellip;': '\u2026',
```

Comment:
would it make sense to include all the characters from the offical spec? 🤔
https://www.w3.org/TR/html4/sgml/entities.html